be/src/exprs/anyval_util.cpp

```diff
@@ -146,6 +147,10 @@ FunctionContext::TypeDesc AnyValUtil::column_type_to_type_desc(const TypeDescrip
     case TYPE_OBJECT:
         out.type = FunctionContext::TYPE_OBJECT;
         // FIXME(cmy): is this fallthrough meaningful?
+        break;    //TODO(weixiang):why no break here?
```

Comment:
If not sure why no `break` here, it's better to not change the original behavior here. 
Avoid introducing weird and unrelated bug by this feature.